src/utils/draw.js

```diff
@@ -4,6 +4,17 @@ function getDateInfo(data, date) {
   return data.contributions.find(contrib => contrib.date === date);
 }
 
+function getContributionCount(graphEntries) {
+  return graphEntries.reduce((rowTotal, row) => {
+    return (
+      rowTotal +
+      row.reduce((colTotal, col) => {
+        return colTotal + (col.info ? col.info.count : 0);
```

Comment:
You can change `=> { return foo; }` to `=> foo`

Reply:
@styfle I usually do, I was in a hurry this time 😂 thanks will do a bit of clean up soon!